cmd/ctr/commands/run/run.go

```diff
@@ -201,7 +207,22 @@ var Command = cli.Command{
 						logrus.WithError(err).Error("network review")
 					}
 				}
-				task.Delete(ctx)
+
+				// task must be stopped before deletion
```

Comment:
While the runc and crun delete commands can delete containers in the created state directly, the runtime-spec defines  `Attempting to delete a container that is not stopped MUST have no effect on the container and MUST generate an error`
> https://github.com/opencontainers/runtime-spec/blob/main/runtime.md#delete